ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -1821,6 +1824,23 @@ enum pkcs11_rc set_key_data(struct obj_attrs **head, void *data,
 	}
 }
 
+enum pkcs11_rc get_key_data_to_wrap(struct pkcs11_object *obj, void **data,
```

Comment:
Now looking this in place -- all neighbors has type `struct obj_attrs **head` and not `struct pkcs11_object *`. So perhaps it would be better to change this one too like that.

Reply:
Changed